apps/workers/db-check/worker_test.go

```diff
@@ -0,0 +1,201 @@
+package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/hydronica/trial"
+	"github.com/jmoiron/sqlx"
+)
+
+func TestGetZeroSums(t *testing.T) {
+	type Row struct {
+		IntCol  int64
+		NumCol  []uint8
+		DateCol time.Time
```

Comment:
to improve readability you may want to use normal types and convert in the test function. 
``` go 
type Row struct {
date string // format of 2006-01-02T15
integer int
numeric string 
} 

```